configs/default/IFRC-IFLIGHT_BLITZ_F405.config

```diff
@@ -0,0 +1,97 @@
+# Betaflight / STM32F405 (S405) 4.2.9 Apr 27 2021 / 19:33:01 (e097f4ab7) MSP API: 1.43
```

Comment:
This is still exported from third-party firmware, not from our official unified target firmware. How many times do you wanna waste on us? How many times do I need to teach you?

Reply:
After a call with @DusKing1 we found out that everything is fine on the version. The config file is exported from official STM32F405 4.2.9 from April 27 19.33 . Looks all good here finally

![WeChat Image_20210823173025](https://user-images.githubusercontent.com/9414972/130424556-62f4a2de-5410-4faa-b51f-d5b34da8f580.png)